netlify.toml

```diff
@@ -8,7 +8,7 @@
   publish= "build/"
 
 [context.deploy-preview.environment]
```

Comment:
Can you revert the changes you made to this file, change the local DEV environment URL to the STAGING environment URL? That is the link to the staging env that is used by Netlify for deploy previews.

Reply:
Done!